lib/lsperfm/core/run.rb

```diff
@@ -18,9 +18,9 @@ class Runner
 
     attr_reader :command
 
-    def initialize(config, debug = false, logstash_home = Dir.pwd)
+    def initialize(config, workers = "1", debug = false, logstash_home = Dir.pwd)
```

Comment:
I understand, I've more tendency to like workers as it, and convert it to string as close to the place that is necessary.